### ISSUE TITLE
ERROR: test_exist_font_config_option

### ISSUE THREAD

**Issue by tk0miya** [blockdiag/blockdiag#66]
After fixing `tox` in https://bitbucket.org/blockdiag/blockdiag/pull-request/12, I've catched this error on Windows. Probably some temp file is not closed before unlinking - Windows is kind of sensitive to that.

```
...
py27 runtests: commands[0] | nosetests
........................................................E.............................................................................................................................
.............SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.S
SS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSSFSSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS
.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS.SSS............................................
======================================================================
ERROR: test_exist_font_config_option (blockdiag.tests.test_boot_params.TestBootParams)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "E:\p\blockdiag\src\blockdiag\tests\test_boot_params.py", line 134, in test_exist_font_config_option
    os.unlink(tmp[1])
WindowsError: [Error 32] The process cannot access the file because it is being used by another process: 'c:\\users\\user\\appdata\\local\\temp\\tmpco0iwh'
```

----------------------------------------
- Bitbucket: https://bitbucket.org/blockdiag/blockdiag/issue/60
- Originally reported by: anatoly techtonik
- Originally created at: 2014-10-28T15:13:17.454

**Comment by tk0miya** [comment]
_From Takeshi KOMIYA on 2014-10-31 11:12:34+00:00_

Fix #60 tempfile is not closed on testcase for -f option

→ \<\<cset 8c435fad775bf4f6ef9d6593074bf580ba124464\>\>

**Comment by tk0miya** [comment]
_From Takeshi KOMIYA on 2014-10-31 11:15:47+00:00_

[anatoly techtonik](https://bitbucket.org/techtonik) The change at 36bf64dad1e6 would resolve the problem.
but I don't have windows environment.
Could you try it please?

Thanks.

**Comment by tk0miya** [comment]
_From anatoly techtonik on 2014-10-31 12:00:21+00:00_

If you can merge https://bitbucket.org/blockdiag/blockdiag/pull-request/12/add-requirementstxt-for-tox-testing/diff - that will make it easier for me to try.

**Comment by tk0miya** [comment]
_From anatoly techtonik on 2014-10-31 12:09:52+00:00_

Yes. It is solved for 1.4.7*
